cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -1098,6 +1098,19 @@ def right_scheduler_schema(self, data, **kwargs):
                     f"Scheduling > *Queues section is not appropriate to the Scheduler: {configured_scheduler}."
                 )
 
+    @validates_schema
+    def same_subnet_in_different_queues(self, data, **kwargs):
+        """Validate subnet_ids configured in different queues are the same."""
+        for queues in ["slurm_queues", "aws_batch_queues"]:
+            if data.get(queues, None):
```

Comment:
Feels more pythonic to use `if queue in data:` here

Reply:
I was using `data.get(queues, None)` to be aligned with the other validation function in our code, now change other places to use `if queues in data` to looks more clear